src/plugins/intel_gpu/src/graph/impls/cm/pa_single_token.cm

```diff
@@ -261,9 +261,9 @@ extern "C" _GENX_MAIN_ void KERNEL_NAME(
     matrix<half, KV_PARTITION_STEP_NUM / 2 * REG_M, REG_K> Pmat = 0;
     #else
         #if Q_RepeatCount != 1
-            matrix<half, Q_RepeatCount, KV_PARTITION_STEP_NUM * REG_M * REG_K> Pmat = 0;
+            matrix<half, Q_RepeatCount, KV_PARTITION_STEP_NUM * REG_M * REG_N> Pmat = 0;
```

Comment:
Currently, only xe2 is supported and for xe2 REG_N==REG_K, it will not impact the kernel result. 
But in the future to support xe, we need distinguish them during online softmax computation.